ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/PlatformApplications.scala

```diff
@@ -46,17 +46,30 @@ object PlatformApplications {
       "Max TTL's granularity is subsecond. Ledger Server does not support subsecond granularity for this configuration - please use whole seconds."
     )
 
-    def getLedgerId: String =
-      ledgerId.getOrElse(
-        throw new IllegalStateException("Attempted to access ledger ID, but none is configured."))
+    @SuppressWarnings(Array("org.wartremover.warts.StringPlusAny"))
+    def assertStaticLedgerId: String =
+      ledgerId match {
+        case RequestedLedgerAPIMode.Static(ledgerId) => ledgerId
+        case _ =>
+          throw new IllegalArgumentException("Unsupported ledger id config: " + ledgerId)
+      }
+
+    def getConfiguredLedgerId: Option[String] = {
```

Comment:
this seems to add little value on top of `ledgerId`, so i'd rather not introduce it.

Reply:
I am converting the only usage of this function to take `RequestedLedgerAPIMode` instead of a somehow-blind `Option[String]`